weave-js/src/components/Panel2/PanelRootBrowser/ProjectObjectsTable.tsx

```diff
@@ -130,3 +131,22 @@ export const ProjectObjectsTable: React.FC<
     </PanelContextProvider>
   );
 };
+
+const EmptyStateComp: FC = () => {
+  return (
+    <EmptyStateContainer>
+      <EmptyStateContent>No objects found</EmptyStateContent>
+    </EmptyStateContainer>
+  );
+};
+
+const EmptyState = memo(EmptyStateComp);
+
+const EmptyStateContainer = styled.div`
+  height: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+`;
+
+const EmptyStateContent = styled.div``;
```

Comment:
optional: I feel like this isn't adding anything, maybe just use `<div>` above?

Reply:
Yeah, this was left over from when I thought we were gonna do something fancier than "No objects found`. Removed.